pkgs/applications/misc/rofimoji/default.nix

```diff
@@ -0,0 +1,43 @@
+{ buildPythonApplication
+, fetchFromGitHub
+, lib
+
+, waylandSupport ? true
+, x11Support ? true
+
+, ConfigArgParse
+, pyxdg
+, rofi
+, wl-clipboard
+, wtype
+, xdotool
+, xsel
+}:
+
+buildPythonApplication rec {
+  pname = "rofimoji";
+  version = "4.3.0";
+
+  src = fetchFromGitHub {
+    owner = "fdw";
+    repo = "rofimoji";
+    rev = version;
+    sha256 = "08ayndpifr04njpijc5n5ii5nvibfpab39p6ngyyj0pb43792a8j";
+  };
+
```

Comment:
I think it would be good to note that everything but `ConfigArgParse` and `pyxdg` are dependencies on binaries that we expect to be added to `PATH` via the python wrapper

Reply:
Does this comment look good?

Reply:
Yep, perfect